lib/Backends/Interpreter/InterpreterFunction.cpp

```diff
@@ -230,7 +229,8 @@ llvm::Error BoundInterpreterFunction::execute(IRFunction *F,
   }
 
   {
-    auto ev = context->scopedEvent("eraseTensors");
```

Comment:
q: why this was replaced by the macro below?

Reply:
To match the style of other files.